Assets/MRTK/Core/Definitions/MixedRealityToolkitExperienceSettingsProfile.asset

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 3193d07953d1c3e44ad7ad26908fe6b2, type: 3}
+  m_Name: MixedRealityToolkitExperienceSettingsProfile
+  m_EditorClassIdentifier: 
+  isCustomProfile: 1
```

Comment:
I am wondering if this value should be 0. Also I am thinking maybe this file should be moved to SDK/Profiles since other default profiles seem to live there.